services/tenant-ui/frontend/src/components/oca/Oca.vue

```diff
@@ -20,7 +20,7 @@
       <template #header>
         <div class="flex justify-content-between">
           <div class="flex justify-content-start">
-            <CreateOca />
+            <!-- <CreateOca v-if="config.frontend.showWritableComponents" /> -->
```

Comment:
Is this supposed to be commented out, rather than just `v-if`'d like all the others?

Reply:
Sorry, my comment was misleading. I would hide it in the "production" view, it can stay as a sandbox/development feature though.

Reply:
My bad.....
K. I'll fix that.